### ISSUE TITLE
ACCES_FINE _LOCATION not accepted by Google Play

### ISSUE THREAD

**Issue by scal-csimon** [jamesmontemagno/GeolocatorPlugin#341]
For common questions and issues see the [FAQ](https://jamesmontemagno.github.io/GeolocatorPlugin/FAQ.html)
## Bug Information

Version Number of Plugin: 4.5.06
Device Tested On: Pixel 3a
Simulator Tested On: Pie 9
Version of VS: 2019
Version of Xamarin: 16
Versions of other things you are using:

### Steps to reproduce the Behavior
Buid your Application with plugin Xam.Plugin.Geolocator

### Expected Behavior
Manifest without ACCESS_FINE_LOCATION
Google does not accept anymore ACCESS_FINE_LOCATION witout good reason.
You should remove automatically add this permission and let developper add it themself if thy want.

Its **CRITICAL** for publish in google play

### Actual Behavior
automatically add 
<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>

**Comment by jamesmontemagno** [comment]
I don't think that is accurate -> https://developer.android.com/training/location/permissions#foreground their documentation tells you to use this.
